src/std/string.md

```diff
@@ -13,7 +13,7 @@ fn main() {
     s2.push('!');
     println!("s2: len = {}, capacity = {}", s2.len(), s2.capacity());
 
-    let s3 = String::from("🇨🇭");
+    let s3 = String::from("A Flag:🇨🇭");
```

Comment:
We need to be able to reason about the string length at a glance in both bytes and characters. So we should not add extra text here:

```suggestion
    let s3 = String::from("🇨🇭");
```